player/command.c

```diff
@@ -2173,6 +2173,8 @@ static int get_track_entry(int item, int action, void *arg, void *ctx)
         {"demux-samplerate", SUB_PROP_INT(p.samplerate),
                         .unavailable = !p.samplerate},
         {"demux-fps",   SUB_PROP_DOUBLE(p.fps), .unavailable = p.fps <= 0},
+        {"demux-bitrate", SUB_PROP_INT(p.bitrate), .unavailable = p.bitrate <= 0},
+        {"demux-rotation", SUB_PROP_INT(p.rotate), .unavailable = p.rotate <= 0},
```

Comment:
is 0 degrees not a valid rotation value?

Reply:
I think of it as an illegal value because I don't want to check whether the track is video. In any case, 0 degrees is indeed a valid value, but it also means no rotation. I am glad to modify it if necessary.

Reply:
Having absence of value mean "no rotation" is fine.